src/components/HeaderLogin/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const HeaderLogin = (props) => (
```

Comment:
You could destructure your props right here:

```js
const HeaderLogin = ({ photo, message }) => (
```

And then use `photo` and `message` instead of `props.photo` and `props.message`.